.tool-versions

```diff
@@ -1 +0,0 @@
-nodejs 12.16.1
```

Comment:
This file is used by [`asdf`](https://asdf-vm.com), and was committed as part of a [larger commit](https://github.com/selfdefined/web-app/commit/beb2f5db69d81fffa92b294f5663624f12783de0) (likely due to a stray `git add .`). `asdf` is a lovely tool, but I don't think it's an actual build dependency of this app. As such, I propose that this file be removed; users of `asdf` (such as myself!) can create a [global `.gitignore`](https://davidwalsh.name/global-gitignore) which prevents this file from being committed to upstream projects.